pkg/run/run.go

```diff
@@ -132,10 +132,10 @@ func (l *localServices) Start() error {
 	// Create a new Worker Pool
 	// TODO: We may want to override GetWorker on the default ProcessPool
 	// For now we'll use the default and expand from there
```

Comment:
can you just delete this? no point having the commented  out code.

Reply:
done